addons/mail/static/src/new/thread/attachment_list.xml

```diff
@@ -27,7 +27,7 @@
                     </div>
                     <div class="position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end flax-wrap flex-column">
                         <div t-if="canDelete(attachment)"
-                            class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover"
+                            class="o-mail-attachment-unlink btn btn-sm btn-dark rounded opacity-75 opacity-100-hover"
```

Comment:
Note that you don't have to introduce the classname. Could have used `aria-label`.

This makes the template noiser, and the tests are also less robust because it relies on technical details that are not visible on UI. `aria-label` is like a visual clue for aria, hence why it's a good candidate